### PR TITLE
upgrade protobuf to 3.1.0

### PR DESCRIPTION
**Pull request by kashif**

### CODE REVIEW

**Comment by kashif** [comment]
For issues #4561 and #4371

**Comment by jart** [comment]
Mr. Jenkins: test this please

**Comment by kashif** [comment]
@jart can you kindly try to run jenkins again?

File: tensorflow/g3doc/get_started/os_setup.md [hunk 1]
```diff
@@ -859,7 +859,7 @@ and you left the Cuda or cuDNN version empty, try specifying them explicitly.
 ### Protobuf library related issues
 
 TensorFlow pip package depends on protobuf pip package version
-3.0.0b2. Protobuf's pip package downloaded from [PyPI](https://pypi.python.org)
+3.0.0. Protobuf's pip package downloaded from [PyPI](https://pypi.python.org)
```
Comment (jhseu) [inline tensorflow/g3doc/get_started/os_setup.md]: Can you change this to 3.1.0?

**Review by jhseu — CHANGES_REQUESTED** [review]
Can you also modify:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/pip_package/setup.py
To change the dependency to 3.1.0?

File: tensorflow/g3doc/get_started/os_setup.md (hunk 1 above)
Comment (kashif) [inline tensorflow/g3doc/get_started/os_setup.md]: as far as I see the protobuf pip package is still at 3.0.0. Or am I missing something?

Comment (jhseu) [inline tensorflow/g3doc/get_started/os_setup.md]: See my other comment about updating:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/pip_package/setup.py

Seems like it's available on pypi to update:
https://pypi.python.org/pypi/protobuf/3.1.0

**Comment by kashif** [comment]
@jhseu done

**Comment by jhseu** [comment]
Jenkins, test this please

**Review by jhseu — APPROVED** [review]

**Comment by jhseu** [comment]
Can you merge or rebase with the latest master? Tests are failing due to being on an old change.

**Comment by jhseu** [comment]
Jenkins, test this please

**Pull request merged**